### PR TITLE
Update dependecies and some fixes for compatibility with the latest npm packages/libraries.

### PR DESCRIPTION
**Pull request by NickMaev**
* Updated dependencies due to deprecation (for compatibility with the latest npm packages/libraries, for ex. [antd](https://github.com/ant-design/ant-design/)).
* Replaced "require" by "import" (for compatibility with EcmaScript module output).
* Added ESM support (for compatibility purposes).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,4 +24,8 @@ package-lock.json
 
 # Editor directories and files
 .idea
-.vscode
\ No newline at end of file
+.vscode
+/.vs/slnx.sqlite
```
Comment (SimonSchick) [inline .gitignore]: Should use `.vs` only

**Comment by SimonSchick** [comment]
So this would be a breaking change, correct?

File: .gitignore (hunk 1 above)
Comment (NickMaev) [reply .gitignore]: Agreed

**Comment by SimonSchick** [comment]
Also I'm curious: What is the use case for `module`?
Iirc node esm modules are still pretty niche.

**Comment by NickMaev** [comment]
> Also I'm curious: What is the use case for `module`?
> Iirc node esm modules are still pretty niche.

Didn't understand you, rephrase please. Where did you see `module` in my code?

**Comment by johnnyreilly** [comment]
Closing due to inactivity

**Pull request closed without merging**